assemblies/assembly_managing-assets/main.adoc

```diff
@@ -13,7 +13,7 @@ include::_artifacts/author-group.adoc[]
 As a process administrator, you can use {CENTRAL} in {PRODUCT} to manage assets, such as rules, business processes, and decision tables.
 
 .Prerequisites
-* {EAP_LONG} 7.2.0 is installed. For details, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
+* {EAP_LONG} {EAP_VERSION} is installed. For details, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html-single/installation_guide/[_{EAP_LONG} {EAP_VERSION} Installation Guide_].
```

Comment:
Suggestion - DM/PAM 7.3 supports EAP 7.2, change the URL from "/7.3/" to "/7.2/"

Reply:
Oh man.. That makes me wonder about my other docs. I forgot about that.